### PR TITLE
External Account Binding for KCert client with ACME

### PR DESCRIPTION
**Pull request by shubham-root**
As discussed in #49 attempting to add EAB account registration to KCert.

### CODE REVIEW

File: Services/RenewalHandler.cs [hunk 1]
```diff
@@ -71,6 +71,7 @@ private async Task<(string KID, string Nonce)> InitAsync(string key, Uri acmeDir
     {
         await _acme.ReadDirectoryAsync(acmeDir);
         var nonce = await _acme.GetNonceAsync();
+        // TODO read environment variable to add modify method call signatures for EAB use case
```
Comment (shubham-root) [inline Services/RenewalHandler.cs:74]: I couldn't figure how the environment variables are used in this project so left it for later.

Comment (nabsul) [reply Services/RenewalHandler.cs:74]: Nice work! You just add new fields to the kcertconfig class. Then just bring that class into this one by adding it the same way as certclient up on line 25 here.

Comment (nabsul) [reply Services/RenewalHandler.cs:74]: ... oh, nevermind about the second part. _cfg is already included here.

Comment (shubham-root) [reply Services/RenewalHandler.cs:74]: Understood. Pushed the changes :)

**Comment by nabsul** [comment]
For testing, you can run kcert locally with `dotnet run`. It will automatically use your kubectl credentials to connect to kubernetes.

Instead of environment variable I use dotnet user secrets: https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets#enable-secret-storage

**Comment by nabsul** [comment]
I thought a little more about testing, and I think that won't work actually. You can tun KCert locally as I described, but I don't think renewals will work properly. You'll need to deploy a test version to try it out.

**Comment by shubham-root** [comment]
Sure thanks for the pointers on testing locally. I was a still seeing outage and incredibly slow response times on ZeroSSL today, have not been able to figure out a different SSL service that could let me quickly set up a test for ACME EAB.
Shall prod around with SSL.com maybe and see if something can come out of there. Shall test before marking the draft PR ready for review.

**Comment by nabsul** [comment]
This feature was completed by @Zegorax and is now merged into main. It will be in the next release after testing is compelted.

**Pull request closed without merging**